crates/bitcoin-da/src/service.rs

```diff
@@ -1043,7 +1046,6 @@ mod tests {
         .expect("Error initialazing BitcoinService");
 
         let da_service = Arc::new(da_service);
-        // da_service.clone().spawn_da_queue(_rx);
```

Comment:
keep this. when we generate test cases we always forget to uncomment this and takes 10 mins to debug :D